Automattic/wp-calypso
# Social: Remove unnecessary social connection feature checks

Related to https://github.com/Automattic/jetpack/pull/38216

## Proposed Changes

* Remove the unnecessary feature checks for social connections

## Why are these changes being made?


*

## Testing Instructions



* Goto `/posts/:site`
* Click on options (3 dots) for any post
* Select "Share" and then click Preview
* Confirm that you see Instagram, Nextdoor and Mastodon services just like any other services.

## Pre-merge Checklist
